content/organization/events/online/index.en.md

```diff
@@ -88,52 +88,50 @@ At the very bottom you should see some buttons -- click on the 'Claim Host' butt
 
 {{< figure src="enter-key.png" alt="Screenshot of the 'enter key' form in the Zoom interface" >}}
 
-### Tips for Running the Zoom meeting
+### Tips for running the Zoom meeting
 
```

Comment:
@acolum, what do you think about mentioning the chapters in a notice box like this: 

```suggestion
{{% notice note %}}
These guidance has been updated based on experiences from the Baltimore, Irvine, NYC, and Santa Barbara chapter guidelines, specifically to address issues with trolls in zoom meetings.
{{% /notice %}}


```

Reply:
@rivaquiroga Thank you for this suggestion, but it's fine - we don't have to mention these chapters. It would be more work that it's worth to get their approval before mentioning them on this page.

Reply:
(So it would be great if we can we have the notice note)

Reply:
fyi @acolum we added this in